assets/src/application/services/apollo/mutations/useEntityMutation.ts

```diff
@@ -49,8 +49,8 @@ const useEntityMutation = (type: EntityType, id?: string): EntityMutation => {
 	 */
 	const getMutationOptions = (mutationType: MutationType, input: MutationInput = {}): OperationVariables => {
 		// e.g. "datetimeMutator"
-		const key = `${type.toLowerCase()}Mutator`;
-		const mutator: Mutator = mutators[key];
+		const key = `${type.toLowerCase()}Mutator` as keyof Mutators;
```

Comment:
:+1:

Reply:
cool... so is this basically saying that any value matching a property name in the Mutators object is acceptable?  Kinda like an enum but not hardcoded values.